spyder/plugins/editor.py

```diff
@@ -651,6 +651,14 @@ def get_plugin_actions(self):
         self.register_shortcut(self.file_switcher_action, context="_",
                                name="File switcher", add_sc_to_tip=True)
 
+        self.symbol_finder_action = create_action(self, _('Symbol finder...'),
+                                            icon=ima.icon('symbol_find'),
+                                            tip=_('Fast symbol search in file'),
+                                            triggered=self.call_symbol_finder,
+                                            context=Qt.ApplicationShortcut)

```

Comment:
Not saying you should change this, just a comment for reference.

```python

# I tend to favor this one for several reasons. 
# It can be automatically performed and enforced by tools like yapf
# it results in more lines of code but ir more readable as each line corresponds 
# to a parameter
self.symbol_finder_action = create_action(
    self,
    _('Symbol finder...'),
    icon=ima.icon('symbol_find'),
    tip=_('Fast symbol search in file'),
    triggered=self.call_symbol_finder,
    context=Qt.ApplicationShortcut)

# or 

self.symbol_finder_action = create_action(self,
                                          _('Symbol finder...'),
                                          icon=ima.icon('symbol_find'),
                                          tip=_('Fast symbol search in file'),
                                          triggered=self.call_symbol_finder,
                                          context=Qt.ApplicationShortcut)
```